app/javascript/components/pages/export-list/container.spec.js

```diff
@@ -96,39 +91,37 @@ describe("<ExportList />", () => {
     }
   });
 
-  beforeEach(() => {
-    ({ component } = setupMountedComponent(ExportList, {}, initialState));
-  });
-
   it("should render a table with three rows", () => {
-    expect(component.find(MUIDataTable).find(TableBodyRow)).to.have.lengthOf(3);
+    mountedComponent(<ExportList />, {}, initialState);
+    expect(screen.getAllByRole("row")).toHaveLength(3);
   });
 
   it("should render <PageContainer>", () => {
-    expect(component.find(PageContainer)).to.have.lengthOf(1);
+    mountedComponent(<ExportList />, {}, initialState);
+    expect(screen.getByTestId("page-container")).toBeInTheDocument(1);
   });
 
   it("should render <PageHeading>", () => {
-    expect(component.find(PageHeading)).to.have.lengthOf(1);
+    mountedComponent(<ExportList />, {}, initialState);
+    expect(screen.getByTestId("page-heading")).toBeInTheDocument();
   });
 
   it("should render <PageContent>", () => {
-    expect(component.find(PageContent)).to.have.lengthOf(1);
+    mountedComponent(<ExportList />, {}, initialState);
+    expect(screen.getByText(/messages.record_list.rows_per_page/i)).toBeInTheDocument();
```

Comment:
The `PageContent` component is not related to pagination. I think you will need a test-id.

Reply:
Done

Reply:
Why was this test removed?

Reply:
Done